.github/workflows/test.yml

```diff
@@ -15,6 +15,16 @@ jobs:
   unit-test:
     runs-on: ubuntu-latest
     steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@v1
+        with:
+          egress-policy: block
+          allowed-endpoints:
+            pipelines.actions.githubusercontent.com:443
+            github.com:443
+            storage.googleapis.com:443
+            api.snapcraft.io:443
```

Comment:
Thanks for trying out `harden-runner`! 

Wanted to mention that `api.snapcraft.io:443` is not needed for workflows. 

It is easier to use the [insights page](https://app.stepsecurity.io/github/DataDog/stratus-red-team/actions/runs/2804590199) (URL is in the [build log in the harden-runner step](https://github.com/DataDog/stratus-red-team/runs/7694263200?check_suite_focus=true#step:3:9)) to copy the allowed endpoints. `api.snapcraft.io:443` is not shown there since it is not needed. 

JFYI - here is the [issue](https://github.com/actions/runner-images/issues/4867) where unnecessary calls to `api.snapcraft.io:443` were reported to GitHub.

Reply:
Thanks a lot for your input! Will adapt as needed